packages/ui-core/src/components/Tile/Tile.test.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Tile, { ITileProps, Theme, Radius, Shadow } from './Tile';
+
+const props: ITileProps = {
+    className: 'test-class',
+    href: 'test-link',
+    theme: Theme.DARK,
+    radius: Radius.ROUNDED,
+    shadowLevel: Shadow.HIGH,
+    onClick: jest.fn(),
+};
+
+describe('<Tile />', () => {
+    describe('snapshots', () => {
+        it('renders component', () => {
+            const wrapper = shallow(<Tile>Some test content</Tile>);
+            expect(wrapper).toMatchSnapshot();
+        });
+
+        it('renders component with props', () => {
+            const wrapper = shallow(
+                <Tile {...props}>Some test content</Tile>
+            );
+            expect(wrapper).toMatchSnapshot();
+        });
+    });
+
+    describe('handlers', () => {
+        it('calls onClick', () => {
+            const handleClick = jest.fn();
+
+            const wrapper = shallow(<Tile onClick={handleClick}>Some test content</Tile>);
+
+            wrapper.simulate('click');
+            expect(handleClick).toBeCalled();
+        });
+
+        it('calls mouseEnter', () => {
```

Comment:
Тест не про вызов mouseEnter, кейс не в этом. Тут тестируется реакция на наведение мыши. 
```suggestion
        it('should render shadow on mouse enter', () => {
```

Ниже тоже самое.